tests/framework/installer/ceph_manifests.go

```diff
@@ -1051,6 +1051,7 @@ rules:
   - batch
   resources:
   - jobs
+  - cronjobs
   verbs:
```

Comment:
In the integration tests, let's add a check that the crash collector pruner job is able to be created successfully. For example:
- In the `GetRookCluster()` method in this test file, configure the new pruner setting
- In the `InstallRook()` method of ceph_installer.go, validate that the cronjob is created. In that method there are other examples of testing for the existence of the pods that may be helpful.

Reply:
I updated the integration test to include the suggested checks, but this caused the Jenkins job to fail. From the output of `tests/scripts/makeTestImages.sh`, it looks like the fail is caused by using the official ceph image instead of one built with the local changes, so it doesn't know how to process the `daysToRetain` field.

Reply:
You'll need to skip this check for the upgrade suite. Those tests will start by installing v1.3, then upgrade to 1.4, etc. The cron job won't be created until the latest is installed. You should be able to skip it with a check like this:
```
  if rookVersion == VersionMaster {
    // wait for the cron job
  } 
```
You'll need to pass the rookVersion from the parent caller method.

Reply:
I believe the other CI failures were intermittent. How about fixing this issue and let's see if there are any more failures.